hw-1/dist/index.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+// 1st task:
+function excercise1() {
+    let greeting;
+    greeting = "Hello World";
+    console.log('\nThe first task, output:');
+    console.log(greeting);
+}
+excercise1();
+// 2nd task:
+function excercise2() {
+    let whileLoopGreeting;
+    let numberOfGreetings;
+    whileLoopGreeting = 'Hello';
+    numberOfGreetings = 3;
+    console.log('\nThe second task, output:');
+    while (numberOfGreetings > 0) {
+        console.log(whileLoopGreeting);
+        numberOfGreetings--;
+    }
+    /*      numberOfGreetings = whileLoopGreeting;
+            Error: "Type 'string' is not assignable to type 'number'.ts(2322)" */
+}
+excercise2();
+// 3d task:
+function excercise3() {
+    let n;
```

Comment:
same here - would be good to have `let n:number;` with type check, as ts adds "any" type for not assigned vars